### DISCUSSION TITLE
La création de nouvelles ressource. (bois,pierres...)

### DISCUSSION THREAD

**Discussion by Voisin1234** [Crownicles/Crownicles#1410] [Nouvelles idées]
Hey bonsoir que dirait vous la création de nouvelles ressources?!

Par exemple, un vendeur pourrait offrir d'échanger ces ressources (bois...) pour une arme ou un objet de notre inventaire.
Les ressources accumulées pourraient permettre au joueur d'acheter une arme ou de l'améliorer.
Il pourrait s'agir d'un nouveau magasin (qui serait mis à jour quotidiennement.) Ou alors dans un nouvel endroit sur la carte.
Exemples des ressources gagnés : Arc= 1bois / épée= 2fer ECT...

2:  OU :
Un évènement ou un forgeron (autres) qui nous demanderait de lui apporter  plusieurs potions pour obtenir une meilleure potion et la même chose à propos des objets.
Si c'est un événement cela permet que ça ne se fasse pas en 2 minutes car il faut déjà rencontrez-le Marchand - Lire la mission- trouvée les potions ou les objets pour ensuite lui apporter. On peut également compliquer la chose en ne laissant le droit a donné qu'un objet a chaque rencontre.

L'idée doit être améliorée, je l'éditerai également en modifiant ou en ajoutant plusieurs choses car ce n'était pas trop l'idée original.

La possibilité qu'une arme ou bouclier se casse.   
- Example : pendant les combats si la personne perd ou alors aléatoirement pendant le combat l'arme peut se casser et le joueur doit la réparer. Comme avec une commande /réparer et celle-ci serait indisponible pendant un certain temps

**Comment by Nargacaura** [comment]
Ça serait pas un peu trop complexe à mettre en place, sachant qu'on a déjà l'argent et les gemmes comme moyens d'acheter des objets? (Et comme si les magasins standard, de la guilde et des missions ne sont pas suffisants par eux-même...)

_De plus, ça risquerait d'augmenter plutôt considérablement la charge du bot si on doit comptabiliser chaque matériau pour chaque joueur..._

**Reply by Voisin1234** [reply]
"Ça ne serait pas un peu trop complexe à mettre en place" Alors Go non? XD (j'ai compris) Et on verra pour le moment je vais apporter plusieurs possibilités

**Comment by Feiryn** [comment]
J'avais une idée similaire : ajouter des ressources obtenables tout le long du jeu (pendant les mini-évents notamment). Chaque lieu pourrait donner des ressources différentes : près de la mer des ressources marines, sur le mont céléstrum des minerais, sur l'île volcanique des objets volcaniques etc.
Les monstres pourraient eux aussi donner des ressources.

Ces ressources permettraient de créer divers items, soit des existants comme des armes, armures, potions ou objets, soit en inventer des nouveaux comme des anneaux, amulettes... Soit pour améliorer des items existants. Par exemple, on pourrait avoir "<Nom de l'item> (amélioré +1)" qui aurait des meilleures stats.

Si on rend ça assez long, ça permettrait d'allogner considérablement le temps de jeu end game et ajouter de nouveaux objectifs
